src/components/SlateEditor/plugins/embed/EditImage.tsx

```diff
@@ -114,33 +120,36 @@ const EditImage = ({ embed, saveEmbedUpdates, setEditModus }: Props) => {
   };
 
   return (
-    <StyledEditorWrapper contentEditable={false}>
-      <Overlay />
-      <FocusTrapReact
-        focusTrapOptions={{
-          onDeactivate: () => {
-            setEditModus(false);
-          },
-          clickOutsideDeactivates: true,
-          escapeDeactivates: true,
-        }}>
-        <StyledEditorContent>
-          <ImageEditor
-            embed={embed}
-            onUpdatedImageSettings={onUpdatedImageSettings}
-            imageUpdates={state.imageUpdates}
-          />
-          <FigureInput
-            caption={state.caption}
-            alt={state.alt}
-            madeChanges={state.madeChanges}
-            onChange={onChange}
-            onAbort={onAbort}
-            onSave={onSave}
-          />
-        </StyledEditorContent>
-      </FocusTrapReact>
-    </StyledEditorWrapper>
+    <Portal isOpened key="imagePortal">
+      <StyledModal
+        onClose={() => setEditModus(false)}
+        isOpen
+        controllable
```

Comment:
Kan isOpen og controllable fjernes? Eventuelt bare en av de? Mistenker at modalen kanskje vil fungere uten

Reply:
Må ha med begge for at modalen skal åpne seg